src/Models/ObjectRelation.php

```
@@ -62,8 +66,12 @@
      * @inheritDoc
      */
     #[Override]
-    public function getRelation(): ?string
+    public function getRelation(): string
     {
+        if ($this->relation === null || $this->relation === '') {
```

PHPStan (2.1.15)

Result of || is always false.


(booleanOr.alwaysFalse)